contracts/SushiYieldSource.sol

```diff
@@ -4,13 +4,14 @@ pragma solidity 0.6.12;
 
 import { IYieldSource } from "@pooltogether/yield-source-interface/contracts/IYieldSource.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
 
 import "./ISushiBar.sol";
 import "./ISushi.sol";
 
 /// @title A pooltogether yield source for sushi token
 /// @author Steffel Fenix
-contract SushiYieldSource is IYieldSource {
+contract SushiYieldSource is IYieldSource, ReentrancyGuard {
```

Comment:
You must initialize the guard

Reply:
There is no need to initialize it when using the non upgradeable version one no?

Reply:
@asselstine there is no need to initialize it when using the non upgradeable version one no?

Reply:
The constructor must be called

Reply:
Done: https://github.com/pooltogether/sushi-pooltogether/pull/10/commits/d8be0a80ea8af97938fe07b83f272c7cce166ec2